vector-im/element-x-android
# Add setting to disable notification for invites

## Type of change


## Content


Add setting to disable notification for invites.
Need SDK release with https://github.com/matrix-org/matrix-rust-sdk/pull/2908

## Motivation and context


Closes #1944

## Screenshots / GIFs

<img width="248" alt="image" src="https://github.com/vector-im/element-x-android/assets/3940906/e2dbd6b2-5276-4c19-8958-cf768bd29c02">

+ recorded ones.



## Tests



- Disable the new option
- Invite the user to a room
- observe that no notification is displayed

## Tested devices

- OS version(s):

## Checklist
